srcpkgs/haskell-language-server/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'haskell-language-server'
+pkgname=haskell-language-server
+version=1.0.0
+revision=1
+archs="x86_64"
```

Comment:
Any problems with i686 or musl?

Reply:
Unsure about i686, but musl fails.

Check the comments in the PR for further insight.

Reply:
> Check the comments in the PR for further insight.

If you are pointing me to "cross fails", please look at README.md on how to do native builds of i686 and x86_64-musl on x86_64 host.

Reply:
> please look at README.md on how to do native builds of i686 and x86_64-musl on x86_64 host.

Thank you for this redirection, it is eye opening.

I've triggered native compiling with `x86_64-musl`, `aarch64` and `i686`.

- [x] `x86_64-musl` compiles.
- [ ] `aarch64` won't let me start compiling.
```sh
$ ./xbps-src -m masterdir-aarch64-musl pkg haskell-language-server
ERROR Failed to execute command /void-packages/xbps-src (Exec format error)
```
- [ ] `i686` has a build error when building Haskell's `Maybe`module. When the site is online, I'll be forwarding the issue to them. Currently the site returns a 502: https://gitlab.haskell.org/